bzero() is a define in the NDK header files (but should probably
not be used at all, bzero as deprecated and then removed in POSIX).

diff -u -r ../vim-7.4.1174/src/osdef1.h.in ./src/osdef1.h.in
--- ../vim-7.4.1174/src/osdef1.h.in	2016-01-24 16:17:03.000000000 -0500
+++ ./src/osdef1.h.in	2016-01-24 17:01:12.816949231 -0500
@@ -70,8 +70,6 @@
 #  endif
 # endif
 #endif
-/* used inside of FD_ZERO macro: */
-extern void	bzero(void *, size_t);
 #ifdef HAVE_SETSID
 extern pid_t	setsid(void);
 #endif
